Spigot-API-Patches/0219-Add-moon-phase-API.patch

```diff
@@ -0,0 +1,65 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@GMail.com>
+Date: Sun, 23 Aug 2020 16:32:03 +0200
+Subject: [PATCH] Add moon phase API
+
+
+diff --git a/src/main/java/io/papermc/paper/world/MoonPhase.java b/src/main/java/io/papermc/paper/world/MoonPhase.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..df05153397b42930cd53d37b30824c7e5f008f7e
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/world/MoonPhase.java
+@@ -0,0 +1,36 @@
++package io.papermc.paper.world;
++
++import org.jetbrains.annotations.NotNull;
++
++import java.util.HashMap;
++import java.util.Map;
++
++public enum MoonPhase {
++    FULL_MOON(0L),
++    WANING_GIBBOUS(1L),
++    LAST_QUARTER(2L),
++    WANING_CRESCENT(3L),
++    NEW_MOON(4L),
```

Comment:
![new moon](https://images.spoilertv.com/cache/movies/twilight-new-moon/posters-and-wallpapers/new-moon-poster-008_FULL.jpg)